lib/SILOptimizer/Mandatory/MoveOnlyAddressCheckerUtils.cpp

```diff
@@ -2458,10 +2497,41 @@ void MoveOnlyAddressCheckerPImpl::rewriteUses(
     FieldSensitiveMultiDefPrunedLiveRange &liveness,
     const FieldSensitivePrunedLivenessBoundary &boundary) {
   LLVM_DEBUG(llvm::dbgs() << "MoveOnlyAddressChecker Rewrite Uses!\n");
-  // First remove all destroy_addr that have not been claimed.
+
+  /// Whether the marked value appeared in a discard statement.
+  const bool isDiscardingContext = !addressUseState.dropDeinitInsts.empty();
```

Comment:
IIRC there really isn't a point in doing a const bool. We (if you look through the code base) generally do not use const with value types like bool.

Reply:
In general, in our code base we do not put constants on types like this. It was definitely against LLVM style for a long time.

Reply:
That being said, given that we need to get stuff in, I am fine letting this through for now.